internal/version.go

```diff
@@ -17,14 +17,13 @@ var (
 )
 
 // FullVersion returns a string of version information.
-func FullVersion(invocationBaseImage string) string {
+func FullVersion() string {
 	res := []string{
 		fmt.Sprintf("Version:               %s", Version),
 		fmt.Sprintf("Git commit:            %s", GitCommit),
 		fmt.Sprintf("Built:                 %s", reformatDate(BuildTime)),
 		fmt.Sprintf("OS/Arch:               %s/%s", runtime.GOOS, runtime.GOARCH),
-		fmt.Sprintf("Experimental:          %s", Experimental),
-		fmt.Sprintf("Invocation Base Image: %s", invocationBaseImage),
+		// +experimental fmt.Sprintf("Experimental:          %s", Experimental),
```

Comment:
We can remove the comment

Reply:
requirement for this is to "Only show experimental field if the Docker App CLI was compiled with the build time experimental flag"

Reply:
actualy experimental flag is set using LDFLAGS in Makefile
didn't know about go LDFLAGS and package variables, reminds me Jenkins' groovy init scripts :P